crates/proof-of-sql/src/sql/proof/verifiable_query_result_test_utility.rs

```diff
@@ -33,20 +27,20 @@ pub fn exercise_verification(
     accessor: &impl TestAccessor<RistrettoPoint>,
     table_ref: TableRef,
 ) {
-    let verification_result = res.clone().verify(expr, accessor, &());
-    assert!(
-        verification_result.is_ok(),
-        "Verification failed: {:?}",
-        verification_result.err()
-    );
+    res.clone()
+        .verify(expr, accessor, &())
+        .expect("Verification failed");
```

Comment:
I'd keep the assert above so that we print the error message which is its real purpose.

Reply:
@JayWhite2357 Sure but the point is precisely to print more details about the error, not "Verification fails".